### PR TITLE
Fix some details to deploy to PRE env

### PR DESCRIPTION
**Pull request by laurajaime**

### CODE REVIEW

File: config/initializers/figaro.rb [hunk 1]
```diff
@@ -2,8 +2,8 @@
 
 env = Rails.env
 keys = %w(SECRET_KEY_BASE)
-keys += %w(DB_DATABASE DB_PASSWORD DB_USERNAME)
 unless env.development? || env.test?
+  keys += %w(DB_DATABASE DB_PASSWORD DB_USERNAME)
```
Comment (tramuntanal) [inline config/initializers/figaro.rb]: why this change? we need to connect to the DB in all environments..

**Review by tramuntanal — CHANGES_REQUESTED** [review]
Just one question to address please

**Comment by laurajaime** [comment]
Closed in favor of https://github.com/CodiTramuntana/decidim-dipgra-app/pull/5
